qt/scientific_interfaces/ISISReflectometry/GUI/Save/SavePresenter.cpp

```diff
@@ -244,12 +244,12 @@ void SavePresenter::saveWorkspaces(std::vector<std::string> const &workspaceName
 /** Saves selected workspaces */
 void SavePresenter::saveSelectedWorkspaces() {
   // Check that at least one workspace has been selected for saving
-  auto workspaceNames = m_view->getSelectedWorkspaces();
-  if (workspaceNames.empty()) {
+  auto wkspNames = m_view->getSelectedWorkspaces();
```

Comment:
Was this intended to be changed? It's not a problem, just seems a bit unrelated to this PR?

Reply:
This one was to fix a CppCheck failure that got flagged